common/ulsr_hil_client.py

```diff
@@ -165,91 +170,95 @@ def hil_free_nodes(nodelist, to_project, hil_client=None):
         project = node_info['project']
 
         if (project == to_project):
-            log_info('HIL release: Node `%s` already in `%s` project, skipping' % (node, to_project))
+            log_info('HIL release: Node `%s` already in `%s` project, skipping' % (
+                node, to_project))
             nodelist.remove(node)
         elif (project is not None and project != MAINTENANCE_PROJECT):
             # if a node was not in the free pool or the maintenance project.
             # we remove it from that project, which puts it in the maintenance pool.
```

Comment:
> Does removing a node from a project automatically moves it to the maintenance pool, rather than a free pool?

Yes, it does so automatically. This feature was added by @Izhmash. So my comment "we remove which puts" is accurate.